man/man5/vdev_id.conf.5

```diff
@@ -103,6 +103,9 @@ taken.  The default is bay.
 \fIid\fR - use the scsi id as the slot number.
 
 \fIlun\fR - use the scsi lun as the slot number.
+
+\fISES\fR - use the SCSI Enclosure Services (SES) enclosure device slot number, as reported by
```

Comment:
Can you expand on this slightly as you did in the commit comment to explain that this is not the preferred setting but is being provided as an option for drivers which don't provide any other stable identifier.

Please also mind the 80 character line limit here and in `cmd/vdev_id/vdev_id`.

Reply:
Done, and now complies with the 80 character line limit, notwithstanding the tension between this and the 8 character (tab) indent level ;-)